docs/docs/manual/cellediting.md

```diff
@@ -0,0 +1,162 @@
+
+### Data-type transforms
+
+As detailed in [Data types](exploring#data-types), OpenRefine recognizes seven data types: string, number, boolean, date, array, error, and null. When you use these transforms, OpenRefine will check to see if the given values can be converted, then both transform the data in the cells (such as “3” as a text string to “3” as a number) and convert the data type on each successfully transformed cell. Cells that cannot be transformed will output the original value and maintain their original data type. 
+
+For example, the following column of strings on the left will transform into the values on the right:
+
+|Input|>|Output|
+|---|---|---|
+|23/12/2019|>|2019-12-23T00:00:00Z|
+|14-10-2015|>|2015-10-14T00:00:00Z|
+|2012 02 16|>|2012-02-16T00:00:00Z|
+|August 2nd 1964|>|1964-08-02T00:00:00Z|
+|today|>|today|
+|never|>|never|
+
+This is based on OpenRefine’s ability to recognize dates with the [`toDate()` function](expressions#dates). 
+
+Clicking the “today” cell and editing its data type manually will convert “today” into a value such as “2020-08-14T00:00:00Z”. Attempting the same data-type change on “never” will give you an error message and refuse to proceed.  
+
+Because these common transforms do not offer the ability to output an error instead of the original cell contents, be careful to look for unconverted and untransformed values. You will see a yellow alert at the top of screen that will tell you how many cells were converted - if this number does not match your current row set, you will need to look for and manually correct the remaining cells. 
+
+You can also convert cells into null values or empty strings. This can be useful if you wish to, for example, erase duplicates that you have identified and are analyzing as a subset. 
+
+## Fill down and blank down
+
+Fill down and blank down are two functions most frequently used when encountering data organized into [records](exploring#row-types-rows-vs-records) - that is, multiple rows associated with one specific entity. 
+
+If you receive information in rows mode and want to convert it to records mode, the easiest way is to sort your first column by the value that you want to use as a unique records key, then blank down all the duplicates in that column. OpenRefine will retain the first unique value and erase the rest. Then you can switch from “Show as rows” to “Show as records” and OpenRefine will convert the data based on the remaining values in the first column. Be careful that your data is sorted properly before you begin blanking down - not just the first column but other columns you may want to have in a certain order. For example, you may have multiple identical entries in the first column, one with a value in the second column and one with an empty cell in the second column. In this case you want the value to come first, so that you can clean up empty rows later, once you blank down. 
```

Comment:
A common error here is not using "Make sort permanent," so it's probably worth mentioning.